### ISSUE TITLE
Oral and gut microbial profiling in periodontitis and Parkinson’s disease

### ISSUE THREAD

**Issue by SvetlanaUP** [waldronlab/BugSigDBcuration#404]
Oral and gut microbial profiling in periodontitis and Parkinson’s disease – Ekin Yay – Journal of Oral Microbiology 
https://www.tandfonline.com/doi/full/10.1080/20002297.2024.2331264

**Comment by Merit25** [comment]
Ibizugbe Merit Osaruese

**Comment by SvetlanaUP** [comment]
Hi @Merit25, have you completed your first contribution https://cunysph.az1.qualtrics.com/jfe/form/SV_1RfUgk72TYmcBHE ?

**Comment by Merit25** [comment]
Yes please, is there a way you could help me confirm? @SvetlanaUP

**Comment by SvetlanaUP** [comment]
@Merit25 assigned to this paper curation, enjoy!

**Comment by AleruDivine** [comment]
Hi @Merit25 I noticed you're yet to add experiments and signatures to this study. If you're facing any difficulties, I'd like to collaborate with you on finding the solutions and completing the curation.

I looked through the study and found comparisons and significant signatures to curate.

I'm @AleruDivine on Slack, feel free to reach out.
Thank you so much!

**Comment by MyleeeA** [comment]
Hi @Merit25 
Trust you are doing good 

Just like @AleruDivine 
If it’s okay, I’ll like to collaborate with you and Divine in completing your study

Looking forward to your response 
Thank you 🙏🏾

**Comment by AleruDivine** [comment]
Hello @Merit25, good evening! I hope you are doing okay. I'm just checking in again to see if you need any help with this study seeing no experiments have been added to the study still. I reached out on slack but got no response yet. I hope everything is okay.🙏

**Comment by SvetlanaUP** [comment]
@MyleeeA @AleruDivine please do curate this paper.

**Comment by MyleeeA** [comment]
Thank you @SvetlanaUP

**Comment by AleruDivine** [comment]
Thank you @SvetlanaUP 🙏

**Comment by AleruDivine** [comment]
Good afternoon @SvetlanaUP, we have started working on the study and making notes as we go through it to better understand the paper before the curation. We talked about the paper and we tried to understand/interpret the figures too. Since we were finishing final applications, we agreed to officially start curation today. 

This is the status of the curation.

Thank you so much!

**Comment by MyleeeA** [comment]
Hi @SvetlanaUP 
We are on it, just like @AleruDivine pointed out
Thank you 🙏🏾

**Comment by SvetlanaUP** [comment]
@AleruDivine @MyleeeA what's the status of this curation? Thanks.

**Comment by AleruDivine** [comment]
Good afternoon @SvetlanaUP,  @MyleeeA and I are close to completing the curation, but she’s currently under the weather. There’s been a weather change in some parts of Nigeria and it is very extreme in the North, where she is at the moment and this has affected her health.  As this is a collaborative effort, we are on a brief hold until she is strong enough to continue. 

Please bear with us 🙏 this will be ready for review soon. We sincerely apologize for the delay.

**Comment by MyleeeA** [comment]
Thank you so much @SvetlanaUP 

I apologize for the delay in the curation

**Comment by AleruDivine** [comment]
Good evening @SvetlanaUP
@MyleeeA and I would like to submit this study for review 
https://bugsigdb.org/Study_1127

Thank you so much!

**Comment by SvetlanaUP** [comment]
@AleruDivine @MyleeeA Please correct all the signatures recorded with Genus part abbreviated, for example:
S_copri stands for Segatella copri etc. etc.

**Comment by SvetlanaUP** [comment]
@AleruDivine simple google search showed:
"mitsuokai bacterium" -> Catenibacterium mitsuokai

**Comment by AleruDivine** [comment]
@SvetlanaUP thank you so much 🙏 this will remove a lot of the flagged signature.

**Comment by MyleeeA** [comment]
Thank you so much @SvetlanaUP 

We didn’t want to make too many assumptions 
We’ll correct the signatures

**Comment by SvetlanaUP** [comment]
@MyleeeA please note I did not point out to assumptions: only typos correction and additional investigation on the species signatures abbreviated.

**Comment by MyleeeA** [comment]
Noted @SvetlanaUP 

Will effect the corrections appropriately 

Thank you 🙏🏾

**Comment by AleruDivine** [comment]
Good day @SvetlanaUP 

We have completed the corrections on the curated signatures, and they are now fully uniform.

Thank you very much!

**Comment by AleruDivine** [comment]
Thank you so much for the review @SvetlanaUP. 
@MyleeeA and I will see to the corrections right away.

**Comment by MyleeeA** [comment]
Hi @SvetlanaUP 

This study is ready for a second review 
@AleruDivine and I have effected the corrections
https://bugsigdb.org/Study_1127

Thank you for your patience 🙏🏾

**Comment by AleruDivine** [comment]
Thank you so much @SvetlanaUP 
Great work @MyleeeA 
Phew!😅

**Comment by MyleeeA** [comment]
Thank you so much @SvetlanaUP 

Great work too @AleruDivine
